js/lib/card.js

```diff
@@ -1,4 +1,5 @@
-export function createCard(data) {
+import { getSections } from '/js/lib/db.js'
+export function createCard(data, index) {
```

Comment:
smart. you are passing the index to be able to change the bookmark. Nicely solved :)